doc/classes/VisualShaderNodeColorOp.xml

```diff
@@ -1,35 +1,98 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="VisualShaderNodeColorOp" inherits="VisualShaderNode" category="Core" version="3.2">
 	<brief_description>
+		A [Color] operator to be used within the visual shader graph.
 	</brief_description>
 	<description>
+		Applies [member operator] to two color inputs.
 	</description>
 	<tutorials>
 	</tutorials>
 	<methods>
 	</methods>
 	<members>
 		<member name="operator" type="int" setter="set_operator" getter="get_operator" enum="VisualShaderNodeColorOp.Operator" default="0">
+			An operator to be applied to the inputs. See [enum Operator] for options.
 		</member>
 	</members>
 	<constants>
 		<constant name="OP_SCREEN" value="0" enum="Operator">
+			Produce a screen effect with the following formula:
+			[codeblock]
+			result = vec3( 1.0 ) - ( vec3( 1.0 ) - a ) * ( vec3( 1.0 ) - b );
```

Comment:
Remove extra spaces in parentheses:
```
vec3(1.0) - (vec3(1.0) - a) * (vec3(1.0) - b);
```
(for GLSL we use the same style guide as for C++)